src/miner.cpp

```diff
@@ -222,8 +220,6 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     }
     int64_t nTime2 = GetTimeMicros();
 
-    LogPrint(BCLog::BENCH, "CreateNewBlock() packages: %.2fms (%d packages, %d updated descendants), validity: %.2fms (total %.2fms)\n", 0.001 * (nTime1 - nTimeStart), nPackagesSelected, nDescendantsUpdated, 0.001 * (nTime2 - nTime1), 0.001 * (nTime2 - nTimeStart));
```

Comment:
This log does not appear by default.

It only appears if you explicitly enable `BENCH` logging by either setting: `-debug=1` or `-debug=bench`   

Default example:
```
CreateNewBlock(): block weight: 944 txs: 0 fees: 0 sigops 400
UpdateTip: new best=07b739b9d554a5c95b358ab6043e72dc6dd3434c920b299d57455d454a802b29 height=2 version=0x32000000 log2_work=10.14083 tx=180 date='2022-09-11 23:35:34' progress=1.000000 cache=0.0MiB(1txo)
AddToWallet 994708d4e8ea1d8ccd2996838438cf338d1b5831febf82bfd185e62b290df96a  new
```

Reply:
This one has not really been much of an issue in the logs so could be left as is.